### PR TITLE
[W5.11][W10-A2] Paul-Henri MIGNOT

### PR DESCRIPTION
**Pull request by phmignot**
Sort command, list sorted by name

### CODE REVIEW

File: test/input.txt [hunk 1]
```diff
@@ -59,9 +59,23 @@
 # test sort command
 ##########################################################
 
-  # should return the addressbook sorted by name
+  # should return the addressbook sorted by name
```
Comment (tohcheryl) [inline test/input.txt]: Good job for adding test cases!

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user.
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/SortCommand.java]: Incorrect header comment. Please think of the developer who will extend your work!

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents an contact information.
+ */
+public class Contact {
+
+    public String value;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:8]: Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.

File: src/seedu/addressbook/data/person/Contact.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof Contact // instanceof handles nulls
+                && this.value.equals(((Contact) other).value)); // state check
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:23]: With this you can equate email with phone number; think how.

**Comment by okkhoy** [comment]
@phmignot 
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

**Review by luca590 — COMMENTED** [review]
I'm leaving a comment to fulfill W5.11

Just in terms of naming things, why was sortUniquePersonList renamed to 'sort' which is a little more ambiguous? Also, I think the spacing between () and { is a little inconsistent

**Pull request closed without merging**